clippy_lints/src/utils/conf.rs

```diff
@@ -196,6 +196,8 @@ define_Conf! {
     (upper_case_acronyms_aggressive: bool = false),
     /// Lint: _CARGO_COMMON_METADATA. For internal testing only, ignores the current `publish` settings in the Cargo manifest.
     (cargo_ignore_publish: bool = false),
+    /// Lint: NONSTANDARD_MACRO_BRACES. Enforce specific macros always use certain braces.
```

Comment:
I think this may warrant some additional docs?

Reply:
Are you thinking of an example or just more info?
This is what I came up with and I added multi-line to the `Conf` macro decl.

```rust
/// Lint: NONSTANDARD_MACRO_BRACES. Enforce the named macros always use the braces specified.
///
/// A `MacroMatcher` looks like this in toml `{ name = "macro_name", brace = "(" }`. If the macro is always going to be used
/// with a full path it should named as such `crate_name::macro_name`.
```

Is there a link to what gets generated by the `Conf` struct?

Reply:
Not sure about the link, but I like your proposed change.